.gitignore

```diff
@@ -26,10 +26,8 @@ evap/localsettings.py
 evap/static/css/evap.css
 evap/static/css/evap.css.map
 evap/static/ts/rendered
-evap/static_collected
 evap/media
 evap/database.sqlite3
-evap/upload
```

Comment:
To me, `evap/media` also looks like something that should have its root in `DATADIR` (at least if it is about uploading e.g. images, which is what I'd currently expect)?

Reply:
I don't think it's used at all, is it?

Reply:
quite possible :D